src/Datasource/ConnectionInterface.php

```diff
@@ -51,6 +85,20 @@ public function config();
     public function transactional(callable $transaction);
 
     /**
+     * Checks if a transaction is running.
+     *
+     * @return bool True if a transaction is running else false.
+     */
+    public function inTransaction();
```

Comment:
The elastic-search plugin for example doesn't implement `inTransaction()` or `rollback()`